_layouts/sidenav.html

```diff
@@ -15,8 +15,8 @@
   <nav aria-label="{{ site.data[page.lang].settings.accessible_labels.secondary_navigation }}" class="position-sticky pin-top">
     <ul class="usa-sidenav">
       <li class="usa-sidenav__item">
-        <!-- TODO: Parent item should not be clickable -->
-        <a href="{{ page.url | prepend: site.baseurl }}" class="usa-current"
+        {% assign section_url = site.data[page.lang].settings.nav[page.sidenav].url | default: page.url %}
+        <a href="{{ section_url | prepend: site.baseurl }}" class="usa-current"
```

Comment:
Not strictly related to the ticket, but just like we're doing with `| prepend: site.baseurl` here, we could do similar to the `<a href="{{ item.url }}">` below to fix the sublinks in Federalist.

Reply:
Will do. I will be very happy if this is the fix for [LG-5864](https://cm-jira.usa.gov/browse/LG-5864).

Reply:
Yeah, there might be a handful of different places we'd need to update to fix all of them, but since Federalist includes a path, we need to make sure to account for it with the `site.baseurl`.